go/inst/instance_dao.go

```diff
@@ -287,9 +287,9 @@ func expectReplicationThreadsState(instanceKey *InstanceKey, expectedState Repli
 	if err != nil {
 		return false, err
 	}
-	err = sqlutils.QueryRowsMap(db, "show slave status", func(m sqlutils.RowMap) error {
-		ioThreadState := ReplicationThreadStateFromStatus(m.GetString("Slave_IO_Running"))
-		sqlThreadState := ReplicationThreadStateFromStatus(m.GetString("Slave_SQL_Running"))
+	err = sqlutils.QueryRowsMap(db, "show subordinate status", func(m sqlutils.RowMap) error {
```

Comment:
I feel like this PR is machine generated and without context to the operation of `orchestrator` nor `mysql`. `show subordinate status` is not a valid command. `Subordinate_IO_Running` is not a value returned by this query. `log_subordinate_updates` is not an actual MySQL variable. etc.